### PR TITLE
Added settings

### PR DESCRIPTION
**Pull request by carlosfruitcup**
Added Volume, Sound, Shadows, Fullscreen, back button.

### CODE REVIEW

**Comment by the-gem-dev** [comment]
Could you remove the assets folder which I gave you from the commit

**Comment by the-gem-dev** [comment]
Take out the files located under the src/2d assets its not suppose to be there. Those are assets we want to use and they should not be included.

**Comment by PawBud** [comment]
Ok, so firstly I can see that you have added Arthuro's commits too?

File: README.md [hunk 1]
```diff
@@ -12,4 +12,4 @@ we are likely to work in teams rather than individually so your performance/cont
 
 * [GDevelop forums](https://forum.gdevelop-app.com) and [Discord chat](https://discord.gg/rjdYHvj).
 * [GDevelop homepage](https://gdevelop-app.com) ([open-source](https://github.com/4ian/GDevelop-website))
-* [GDevelop wiki (documentation)](http://wiki.compilgames.net)
\ No newline at end of file
+* [GDevelop wiki (documentation)](http://wiki.compilgames.net)
```
Comment (PawBud) [inline README.md:15]: I am pretty much sure that this needs to be `http://wiki.compilgames.net/doku.php/gdevelop5/start`. Rest looks good :)

Comment (PawBud) [reply README.md:15]: Update Remove this line; I will fix it myself, Sorry for the confusion :/.

**Pull request merged**